tests-with-openai/test_data/evaluation_1.yaml

```diff
@@ -36,6 +38,31 @@
     1. **OSLO T2** (IRI: `urn:uuid:f1769de8-9aeb-11e5-91da-b8763fd99c5f`)
     2. **OSLO T1** (IRI: `urn:uuid:f1769dd6-9aeb-11e5-91da-b8763fd99c5f`)
   actual_steps:
+  - name: retrieval
+    id: call_3
+    args:
+      query: transformers Substation OSLO
+      k: 2
+    status: success
+    output: |-
+      [
+        {
+          "id": 1,
+          "text": "Transformer OSLO T1 is in Substation Oslo."
+        },
+        {
+          "id": 2,
+          "text": "Transformer OSLO T2 is in Substation Oslo."
```

Comment:
Should we have strings here for the ids instead of integers? Should we provide IRIs?

Reply:
Replaced by IRIs